tests/integration/components/profile-test.js

```diff
@@ -0,0 +1,43 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, settled } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | profile-component', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('button appearance based on isDev property', async function (assert) {
+    this.set('handleShowEditProfilePictureModal', function () {
+      //
+    });
```

Comment:
Why this is empty 
If this is not necessary can you  remove this

Reply:
@vinit717  hy I was also testing the handleShowEditProfilePictureModal function. as you mentioned it should not be empty I've made changes accordingly. please review it thanks.